[Azure Log Analytics] Remove server_url default value (#35123)
demisto/content: Packs/AzureLogAnalytics/ReleaseNotes/1_1_35.md

#### Integrations

##### Azure Log Analytics

> - Fixed an issue where setting the ***Azure Cloud*** parameter with "GCC High" value, the integration would failed to run commands.

Reviewer: ```suggestion
- Fixed an issue where the ***Azure Cloud*** parameter was not handled correctly.
```
